### ISSUE TITLE
Add information about optimal placing of tubes for HVAC filter

### ISSUE THREAD

**Issue by ForceConstant** [gcormier/esphome-pressure#8]
I would like to use this device for measuring health of HVAC filter, and don't know anything about where the tubes should be located. i.e. does it have to be right in front/behind filter? Does it have to be in center, or can it be just off the edge, etc?

**Comment by gcormier** [comment]
There is optimal placement with pitot tubes to minimize the effect of the airflow on the pressure reading.  eg https://www.amazon.ca/Fieldpiece-Static-Pressure-Probes-Manometers/dp/B0039UHM9U

That said, I think as long as they are in the same place you're hopefully going to get similar readings to see the relative information.

If someone more qualified wants to contribute info, happy to merge.

**Comment by drew-hill** [comment]
Love your tubing suggestions. Do you have barbed fittings that you like for connecting the unit to an air handler?

**Comment by gcormier** [comment]
Nope. Personally I just jam my tubes in the ducts. I find the pitot tubes brutally expensive for what they are.

**Comment by drew-hill** [comment]
> Nope. Personally I just jam my tubes in the ducts. I find the pitot tubes brutally expensive for what they are.

Hah! Do they seem to work reasonably well that way? I've found some magnetic pitot tubes on amazon for ~$13 USD in the US.

**Comment by gcormier** [comment]
That's a good price! Let me know how it goes and we can update some info to share as well. It seems to work fine for me but I don't have a comparison. I do have some pitot tubes from my manometer so I could try those sometime if I dig them out.

**Comment by Cellivar** [comment]
Sliding in with some updated information I've learned. For filter pressure differential in particularly we're looking for static pressure. A pitot tube is for dynamic pressure and isn't useful for just filter loading. [A good thread.](https://sawmillcreek.org/showthread.php?293302-Pitot-Tube-Location-for-Filter-Monitoring) Some specific nuggets:

> To improve the accuracy and account for any flow/pressure changes on the exit side downstream of the filter you can connect the other leg of the manometer to the exit of your filter. This is a differential pressure measurement.

> A rule of thumb is no direction or flow disturbance 10 pipe diameters (hydraulic diameter for non-round) prior to the measurement point t and 5 pipe diameters past the measurement point.
In the end you’re just looking to get a relative measurement and much less than these ideal conditions needed. Try to use these distances as a rough estimate but not at the expense/effort of changing things drastically.

> Please note it is perpendicular meaning don’t put a tube or port facing the flow stream. Just put it in the side wall of the pipe and not projected into the pipe. There are inaccuracies that can happen from how you put the port in. The smaller the better with minimal to no intrusions into the flow stream.

> ASHRAE spec requires that [a static pressure probe port on a duct] be flush with the wall.

> I backed the fitting out as far as I could and still maintain a seal. Turns out it does matter. The reading went up exactly 0.2".

> fwiw, relative flow is proportional to the squareroot of one minus the ratio of the change in pressure to the total internal fan pressure. So let's say for a give system configuration your filter pressure changes from .3" to 3.3". IRRC, that DC has an internal pressure of about 15" so 1-3/15 = .8 and the squareroot of .8 is about .9, so about a 10% reduction for a 3" increase. You would not likely notice a 10% change in flow when it happens gradually over time.

> One thing to be aware of with measuring the pressure drop across the filter is that the resistance in the rest of the flow network affects flow and thus the measurement. You want to take a reading at some reference condition most likely with all blast gates open. This condition gives the greatest flow and greatest pressure drop across the filter. You take a reading with filter clean (new is good).

My takeaway is my differential probe installation should be two flush ports on either side of my filter on the ductwork, preferably 3 feet away from the filter or other changes to airflow (within practical limits). I will take a baseline measurement with a clean filter and all vents wide open, a second with the vents set normally, and will compare those readings to measurement over time to determine filter loading.
